docs/APISocketOptions.md

```diff
@@ -867,20 +867,24 @@ For details, see [Packet Filtering & FEC](packet-filtering-and-fec.md).
 | `SRTO_PASSPHRASE`    | 0.0.0 | pre     | `string`   |         | ""       |[10..79]| W   | GSD    |
 
 Sets the passphrase for encryption. This enables encryption on this party (or
-disables it, if an empty passphrase is passed).
+disables it, if an empty passphrase is passed). The password must be minimum
+10 and maximum 79 characters long. If an empty password is specified (default),
+the encryption is disabled.
```

Comment:
The sentence "If an empty password is specified (default),
the encryption is disabled." is redundant (see line 870)

Reply:
Right. How I overlooked it. Will be removed.